src/utils/ItemLocaleUtils.ts

```diff
@@ -0,0 +1,20 @@
+import {localizedLanguagesStrings} from '../i18n';
+import {CarouselDataItem} from '../screens/HomeScreen/components';
+import {DeviceUtils} from './DeviceUtils';
+
+const localizedName = (item: CarouselDataItem): string => {
```

Comment:
Utils are suppose to be generic. Why are we adding util specific for CarouselDataItem? And why do we have 2 functions (localizedName & localizedButtonText) doing the same thing but on different items? merge the two functions into one. And the input to the function should be just the object with type LocalizedLanguagesStrings

Reply:
Fixed,  And created a common function.